napari/_qt/qt_main_window.py

```diff
@@ -435,15 +443,14 @@ class Window:
     def __init__(self, viewer: 'Viewer', *, show: bool = True):
         # create QApplication if it doesn't already exist
         get_app()
```

Comment:
we have two `get_app` functions in the napari codebase now. One for creating QApplication and one for `app-model` Application. Based on the context, we could distinguish it, but I prefer to change it.

I wrote here as I meet such confusion during the review of this PR.

Reply:
That's already a Todo item in https://github.com/napari/napari/issues/4860